benchmarks/nnls/solvers/nnls_scipy.py

```diff
@@ -0,0 +1,24 @@
+from benchopt.base import BaseSolver
+from benchopt.util import safe_import
+
+
+with safe_import() as solver_import:
+    from scipy import optimize
+
+
+class Solver(BaseSolver):
+    name = 'scipy'
+
+    install_cmd = 'pip'
+    requirements = ['scipy']
+
+    def set_objective(self, X, y):
+        self.X, self.y = X, y
+
+    def run(self, n_iter):
+        # XXX HACK scipy: when maxiter < 10 could get a
+        # RuntimeError: too many iterations
+        self.w = optimize.nnls(self.X, self.y, maxiter=n_iter + 10)[0]
```

Comment:
I still get some RuntimeError 

looking at scipy nnls fortran code it seems it runs without taking into account maxiter and if in the end it required more iterations than maxiter to converge you get an error. I would explain why all nnls solutions you get are at floating point accuracy whatever the choice of maxiter.

Reply:
so I will simply drop the maxiter option...